helper/general_objects/models.go

```diff
@@ -20,7 +20,7 @@ type Pageable struct {
 	Unpaged    bool   `json:"unpaged"`
 }
 
-type Tag struct {
+type KeyValue struct {
```

Comment:
renaming as we use the same structure for transmission metadata (which are not Tags)